lib/ecs_deploy/capistrano.rb

```diff
@@ -21,25 +21,35 @@
     if fetch(:ecs_tasks)
       regions = Array(fetch(:ecs_region))
       regions = [nil] if regions.empty?
```

Comment:
[imo] `regions = [EcsDeploy.config.default_region || ENV["AWS_DEFAULT_REGION"]] if regions.empty?`

instead of `region = r || EcsDeploy.config.default_region || ENV["AWS_DEFAULT_REGION"]`.